frontend/src/assets/base.css

```diff
@@ -71,3 +71,7 @@ body {
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
 }
+
+.p-datatable{
+  font-size: 15px;
+}
```

Comment:
Not sure if we want to be doing this. This feels a bit like a magic number.

Reply:
I think it's fine (would use `rem` setting over px though since it's the main text on the page) if the UX decision is to have the table font be specifically different. Otherwise I would say it should be the main body font size since it's kind of the main content of the app?